rsts/deployment/deployment/multicluster.rst

```diff
@@ -24,104 +24,289 @@ Scaling Beyond Kubernetes
+
+.. note:: 
+
+   This step is needed so the ``flytepropeller`` instance in the data plane cluster is able to send notifications
+   back to the ``flyteadmin`` service in the control plane.
+
+3. Install Flyte data plane Helm chart:
+
+.. note:: 
+
+   Use the same ``values-eks.yaml`` or ``values-gcp.yaml`` file you used to deploy the controlplane.
 
 .. tabbed:: AWS
 
     .. code-block::
 
-        helm upgrade flyte -n flyte flyteorg/flyte-core values.yaml \
-            -f values-eks.yaml \
-            -f values-dataplane.yaml \
-            --create-namespace flyte --install
+        helm install flyte-core-data flyteorg/flyte-core -n flyte \ 
+        --values values-eks.yaml --values values-dataplane.yaml \ 
+        --create-namespace
 
 .. tabbed:: GCP
 
     .. code-block::
 
-        helm upgrade flyte -n flyte flyteorg/flyte-core values.yaml \
-            -f values-gcp.yaml \
-            -f values-dataplane.yaml \
-            --create-namespace flyte --install
+        helm install flyte-core-data -n flyte flyteorg/flyte-core  \
+            --values values-gcp.yaml \
+            --values values-dataplane.yaml \
+            --create-namespace flyte 
 
+.. _control-plane-deployment:
 
-User and Control Plane Deployment
+Control Plane configuration
 *********************************
 
-Some Flyte deployments may choose to run the control plane separate from the data
-plane. FlyteAdmin is designed to create Kubernetes resources in one or more
-Flyte data plane clusters. For the admin to access remote clusters, it needs
-credentials to each cluster.
+For ``flyteadmin`` to access and create Kubernetes resources in one or more
+Flyte data plane clusters , it needs credentials to each cluster.
+Flyte makes use of Kubernetess Service Accounts to enable every data plane cluster to perform
+authenticated requests to the Kubernetes API Server.
+The default behaviour is that ``flyteadmin`` creates a `ServiceAccount <https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/templates/admin/rbac.yaml#L4>`_
+in each data plane cluster. 
+In order to verify requests, the Kubernetes API Server expects a `signed bearer token <https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens>`__
+attached to the Service Account. As of Kubernetes 1.24 and above, the bearer token has to be generated manually.
 
-In Kubernetes, scoped service credentials are created by configuring a "Role"
-resource in a Kubernetes cluster. When you attach the role to a "ServiceAccount",
-Kubernetes generates a bearer token that permits access. Hence, create a
-FlyteAdmin `ServiceAccount <https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/templates/admin/rbac.yaml#L4>`_
-in each data plane cluster to generate these tokens.
 
-.. warning::
-  
-   **Never delete a ServiceAccount 🛑**
+1. Use the following manifest to create a long-lived bearer token for the ``flyteadmin`` Service Account in your dataplane cluster:
```

Comment:
We should add this to the deployment charts and not ask the users to create this.
The user should only retrieve it and update the secrets in the control plane (which we should also create in the deployment chart)